app/forms/decidim/participatory_documents/suggestion_form.rb

```diff
@@ -6,29 +6,25 @@ class SuggestionForm < Decidim::Form
       include Decidim::AttachmentAttributes
       include Decidim::HasUploadValidations
 
-      MIN_BODY_LENGTH = 15
-      MAX_BODY_LENGTH = 150
-
       attribute :body, String
       attribute :file
 
+      min_length = Decidim::ParticipatoryDocuments.config.min_suggestion_length
+      max_length = Decidim::ParticipatoryDocuments.config.max_suggestion_length
+
```

Comment:
I would prefere not to have variable assignation here. Anyway in here we only are checking for the default values of the module, not for the assigned component settings (also missing from the tests).

better to add the `current_compoent` context and get in a private method the appropiate length:

```ruby
private

def min_length
  current_component.settings.min_suggestion_length || Decidim::ParticipatoryDocuments.config.min_suggestion_length
end